engines/kgs-bot/config.txt

```diff
@@ -0,0 +1,5 @@
+mode=custom
+rules=chinese
```

Comment:
maybe `newzealand` instead because it allows multi-stone suicide? https://www.gokgs.com/help/rulesets.html 
suicide moves are allowed throughout the training of our adversary

Reply:
Sure, changed